mw-webapp/src/component/themeSwitcher/ThemeSwitcher.tsx

```diff
@@ -78,6 +78,7 @@ export const ThemeSwitcher = (props: ThemeSwitcherProps) => {
         className={clsx(styles.iconWrapper, props.className)}
         onClick={onChangeTheme}
         data-cy={props.dataCy}
+        aria-label={props.dataCy}
```

Comment:
We will probably remove data-attriutes for testing purposes from production some day

Reply:
Answered in dialogue with Ekaterina